src/components/accessories/billTable/BillTable.tsx

```diff
@@ -245,23 +257,46 @@ export const BillTable: FC<IBillTableProps> = ({
                     onChange={dateFieldHandleOnChange("toDate")}
                   />
                 </div>
-              </div>
-              <div className="row start-sm center-xs">
-                <div className="fullWidth filterBillForm__item">
-                  <PatientAutocomplete
+                <div className="filterBillForm__item">
+                  <DateField
+                    fieldName="month"
+                    views={["month"]}
+                    fieldValue={formik.values.month}
+                    disableFuture={true}
+                    theme="regular"
+                    format="MMMM"
+                    isValid={isValid("month")}
+                    errorText={getErrorText("month")}
+                    label={t("bill.month")}
+                    onChange={dateFieldHandleOnChange("month")}
+                  />
+                </div>
+                <div className="filterBillForm__item">
+                  <DateField
+                    fieldName="year"
+                    views={["year"]}
+                    fieldValue={formik.values.year}
+                    disableFuture={true}
+                    theme="regular"
+                    format="yyyy"
+                    isValid={isValid("year")}
+                    errorText={getErrorText("year")}
+                    label={t("bill.year")}
+                    onChange={dateFieldHandleOnChange("year")}
+                  />
```

Comment:
![image](https://user-images.githubusercontent.com/8697550/145431888-f4e436d0-230c-493c-a9be-a1f6bf46befc.png)

What is the puropose of this fields when we already have the date from and the date to filter?

Reply:
I rely on the Marco design. But it is just a shortcut to set  fromdate and todate values. I aggree with you that these fields are not so useful. But as I am applying the marco design, it will be good to discuss with Ilario for that.

Reply:
> ![image](https://user-images.githubusercontent.com/8697550/145431704-b275eb72-5d82-49ef-a880-052b839bef5f.png)
> 
> This happens when I try to edit the date field manually

Ok I will fix it.

Reply:
> I rely on the Marco design. But it is just a shortcut to set fromdate and todate values. I aggree with you that these fields are not so useful. But as I am applying the marco design, it will be good to discuss with Ilario for that.

For now just disabled those two fields and focus on the rest